package.json

```diff
@@ -26,7 +26,9 @@
     "serve-dist-oauth": "lerna run --stream serve-dist-oauth --scope @cumulus/api",
     "serve-dist-remote": "lerna run --stream serve-dist-remote --scope @cumulus/api",
     "coverage": "nyc report --reporter text-summary --reporter html",
-    "coveralls": "nyc report --reporter=text-lcov --temp-direcotry=\"./.final_nyc_output\" | coveralls"
+    "coveralls": "nyc report --reporter=text-lcov --temp-direcotry=\"./.final_nyc_output\" | coveralls",
+    "start-unit-test-stack": "SSH_USERS=user:$(id -u):$(id -u) && docker-compose -f ./bamboo/docker-compose.yml -f ./bamboo/docker-compose-local.yml -p cumulusstack up",
```

Comment:
Hmm, just ran into an issue in a new shell where this didn't work as expected, it looks like SSH_USERS isn't being set as expected.